### PR TITLE
fix: read id from hash param only when we we never read it or maintai…

### PR DESCRIPTION
**Pull request by megheaiulian**
…nSelection is true

### CODE REVIEW

File: cosmoz-data-nav.js [hunk 1]
```diff
@@ -973,16 +973,21 @@ class CosmozDataNav extends hauntedPolymer('haunted', useDataNav)(PolymerElement
...
+
 		const path = ['_routeHashParams', hashParam],
 			hashValue = this.get(path),
 			selection = this.items.findIndex(i => this._getItemId(i) === hashValue);
 
 		if (selection < 0 || selection === this.selected) {
 			return;
 		}
-
+		this._readFromHash = true;
```
Comment (cristinecula) [inline cosmoz-data-nav.js]: Maybe a better name?

```suggestion
		this._readFromHashOnce = true;
```

**Review by cristinecula — APPROVED** [review]
Looks good to me.

**Review by nomego — APPROVED** [review]
You know I hate flags but sure :)

File: cosmoz-data-nav.js (hunk 1 above)
Comment (megheaiulian) [reply cosmoz-data-nav.js]: Done!

**Pull request merged**